routes/web.php

```diff
@@ -73,6 +73,14 @@
     Route::name('wiki_bibles.one')->get('/wiki/bibles/{id}', 'WikiController@bible');
     Route::name('wiki_bibles.all')->get('/wiki/bibles',      'WikiController@bibles');
 
+    Route::name('api-key.login')->match(['get', 'post'], 'api-key/login', 'User\UsersController@adminLogin');
```

Comment:
the normal pattern for route name is either (a) camel_case or (b) dot notation.  Please change this (and all others) to 
Route::name('api_key.login')->match(['get', 'post'], 'api_key/login', 'User\UsersController@adminLogin');